### PR TITLE
@hoodie/server

### LINKED ISSUE #534: move hapi logic to @hoodie/server
follow up for #529
- [x] make `@hoodie/server` a hapi plugin that combines all server parts of the core plugins (https://github.com/hoodiehq/hoodie-server/releases/tag/v20.0.0)
- [x] keep in `hoodie`:
  - [x] routes for `/hoodie/client.js`
  - [x] logging with `npmlog` (`@hoodie/server` should not use `npmlog`, instead use `server.log`)
  - [x] fallback UI from `public/` folder
  - [x] admin dashboard
  - [x] hapi-cors-headers
  - [x] hoodie-local-maybe-force-gzip

### PR DESCRIPTION
**Pull request by gr2m**
closes #534

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,41 +14,32 @@
...
     "node-emoji": "^1.3.0",
     "npmlog": "^4.0.0",
-    "pouchdb": "^5.3.2",
-    "pouchdb-admins": "^1.0.1",
-    "pouchdb-users": "^1.0.3",
-    "randomstring": "^1.1.4",
+    "pouchdb-adapter-http": "^5.4.5",
+    "pouchdb-adapter-leveldb": "^5.4.5",
+    "pouchdb-adapter-memory": "^5.4.5",
+    "pouchdb-core": "^5.4.5",
```
Comment (gr2m) [inline package.json:33]: this feels so good. All hail @nolanlawson! Can’t wait to drop `leveldb` in favor of [pouchdb-adapter-fs](https://www.npmjs.com/package/pouchdb-adapter-fs) – no more native builds during install \o/

**Comment by gr2m** [comment]
This is ready for review @hoodiehq/maintainers 

It is a non-breaking change. I’ve tested it with the integration tests of `hoodie-app-tracker` with both levelDB and CouchDB backends.

**Comment by jameswestnz** [comment]
Works locally, LGTM!

**Comment by gr2m** [comment]
let’s ship it LGTM

**Pull request merged**